server/index.js

```diff
@@ -150,6 +151,17 @@ app.get("/api/history/:symbol", async (req, res) => {
   }
 });
 
+app.get("/api/peers/:symbol", async (req, res) => {
+  console.log("app.get peers");
+  const symbol = req.params.symbol;
+  try {
+    const peersData = await fetchWrapper(iex.peers, symbol, 5);
```

Comment:
I think this 5 is doing nothing, can we remove it?

Reply:
Yeah I'm actually not sure what that last argument does, I just took it from a different await fetchWrapper call you had. I'll remove it.